tests/integ/test_horovod.py

```diff
@@ -91,6 +91,7 @@ def _create_and_fit_estimator(sagemaker_session, tf_version, py_version, instanc
         py_version=py_version,
         framework_version=tf_version,
         distribution={"mpi": {"enabled": True}},
+        disable_profiler=True,
```

Comment:
Why is this change required?

Reply:
These three tests often fail in a few small regions with InternalServiceErrors in our release pipeline. Service log showed in one instance the error happened when the training job trying to upload profiler data to s3. We are trying to see if this will fix our tests while the service team tries to root cause the issue.